### PR TITLE
fixups for kernel import

### PR DESCRIPTION
**Pull request by mslacken**
- fixed kernel import from container
- need valid url or dir to import something

### CODE REVIEW

File: internal/pkg/kernel/kernel.go [hunk 1]
```diff
@@ -111,7 +111,7 @@ func ListKernels() ([]string, error) {
 
 func Build(kernelVersion string, kernelName string, root string) (string, error) {
 	kernelDrivers := path.Join(root, "/lib/modules/"+kernelVersion)
-  kernelDriversRelative := path.Join("/lib/modules/"+kernelVersion)
+	kernelDriversRelative := path.Join("/lib/modules/" + kernelVersion)
```
Comment (jcsiadal) [inline internal/pkg/kernel/kernel.go:114]: This should be path.Join("/lib/modules", kernelVersion). The use of Join circumvents the need to cat the filename to the main path.

**Comment by gmkurtzer** [comment]
This looks good to me!

File: internal/pkg/kernel/kernel.go (hunk 1 above)
Comment (gmkurtzer) [reply internal/pkg/kernel/kernel.go:114]: I will fix this in another PR as I think it was my error to begin with. Great catch!

Comment (gmkurtzer) [reply internal/pkg/kernel/kernel.go:114]: addressed in #243.

**Pull request merged**